test/IDE/dump_swift_lookup_tables_objc.swift

```diff
@@ -14,11 +14,12 @@
 // CHECK-NOT: lookup table
 // CHECK:   NSObject:
 // CHECK-NEXT:     TU: NSObject
-// CHECK-NEXT:   NSObjectProtocol:
+// CHECK:   __NSObjectProtocol:
```

Comment:
Where is the extra "Protocol" coming from? Does that kick in before the SwiftPrivate part?